server.py

```diff
@@ -12,6 +13,22 @@
 DEBUG = (os.getenv('DEBUG', 'False') == 'True')
 PORT = os.getenv('VCAP_APP_PORT', '5000')
 
+# Configure Swagger before initilaizing it
+app.config['SWAGGER'] = {
+    "swagger_version": "2.0",
+    "specs": [
+        {
+            "version": "1.0.0",
+            "title": "Wishlist App",
+            "description": "The wishlists resource allow customers to create a collection of products that they wish they had the money to purchase.",
+            "endpoint": 'v1_spec',
+            "route": '/v1/spec'
```

Comment:
do we need/want `/v1` before `/spec` just wondering as we don't use versioning anywhere else

Reply:
I think the professor siad it was a good idea to include versioning in the urls.

Reply:
said*

Reply:
I copied from sample code. Keep it just in case.